include/z64actor.h

```diff
@@ -396,8 +396,8 @@ typedef struct ActorContext {
 } ActorContext; // size = 0x284
 
 typedef enum {
-    /* 00 */ ACTOR_DRAW_DMGEFF_FIRE,
-    /* 01 */ ACTOR_DRAW_DMGEFF_BLUE_FIRE,
+    /*  0 */ ACTOR_DRAW_DMGEFF_FIRE,
+    /*  1 */ ACTOR_DRAW_DMGEFF_BLUE_FIRE,
     /* 10 */ ACTOR_DRAW_DMGEFF_FROZEN_NO_SFX = 10,
     /* 11 */ ACTOR_DRAW_DMGEFF_FROZEN_SFX,
     /* 20 */ ACTOR_DRAW_DMGEFF_LIGHT_ORBS = 20,
```

Comment:
These should be in hex anyway.

Reply:
The reason I put them in decimal is because each category of effect is broken up into 10s in decimal:

0X -> fire
1X -> ice
2X -> light
3X -> sparks

You lose that connection in hex